PATCH-P0-FEATURE-SUSE: Tests in $(datadir)

By default tests would get installed to /usr and as we want it LSB compilant,
we install them into /usr/share.

Maintainer: Michal Hrusecky <Michal.Hrusecky@opensuse.org>

Index: mysql-test/Makefile.am
===================================================================
--- mysql-test/Makefile.am.orig
+++ mysql-test/Makefile.am
@@ -18,7 +18,7 @@
 
 ## Process this file with automake to create Makefile.in
 
-testroot =	$(prefix)
+testroot =	$(datadir)
 testdir =	$(testroot)/mysql-test
 
 test_SCRIPTS =	mtr \
Index: mysql-test/lib/My/SafeProcess/Makefile.am
===================================================================
--- mysql-test/lib/My/SafeProcess/Makefile.am.orig
+++ mysql-test/lib/My/SafeProcess/Makefile.am
@@ -14,7 +14,7 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301  USA
 
-testroot = $(prefix)
+testroot = $(datadir)
 safedir = $(testroot)/mysql-test/lib/My/SafeProcess
 #nobase_bin_PROGRAMS = ...
 safe_PROGRAMS =		my_safe_process
